Extensions/AnchorBehavior/anchorruntimebehavior.js

```diff
@@ -148,9 +148,9 @@ gdjs.AnchorRuntimeBehavior.prototype.doStepPreEvents = function(runtimeScene) {
 
         //Move and resize the object according to the anchors
         if(this._rightEdgeAnchor !== gdjs.AnchorRuntimeBehavior.HorizontalAnchor.NONE)
-            this.owner.setWidth(bottomRightCoord[0] - topLeftCoord[0]);
+            this.owner.setWidth(bottomRightCoord[0]- this.owner.getX() - topLeftCoord[0]);
         if(this._bottomEdgeAnchor !== gdjs.AnchorRuntimeBehavior.VerticalAnchor.NONE)
-            this.owner.setHeight(bottomRightCoord[1] - topLeftCoord[1]);
+            this.owner.setHeight(bottomRightCoord[1]- this.owner.getY() - topLeftCoord[1]);
```

Comment:
@Bouh I see you closed this PR, but actually these 2 lines were fixing an issue! 
We should probably have a new PR with just this? (maybe re-test this, but I think it was useful)

Reply:
@arthuro555  there are thes lines to test again, if it doesn't change anything i'll close this PR.

Reply:
Let me know and feel free to adapt or open the PR if these are useful :)

Reply:
After an test it's no longer necessary.
I close.